src/main/java/it/pagopa/pm/gateway/controller/XPayPaymentController.java

```diff
@@ -42,9 +42,9 @@
 @RequestMapping(REQUEST_PAYMENTS_XPAY)
 public class XPayPaymentController {
 
-    private static final String APP_ORIGIN = "APP";
-    private static final String WEB_ORIGIN = "WEB";
-    private static final List<String> VALID_CLIENT_ID = Arrays.asList(APP_ORIGIN, WEB_ORIGIN);
+    private static final String ECOMMERCE_APP_ORIGIN = "ECOMMERCE_APP";
+    private static final String ECOMMERCE_WEB_ORIGIN = "ECOMMERCE_WEB";
```

Comment:
I can't remember, was there an open point on the client naming?

Reply:
It's solved.